### PR TITLE
[keyman.com] Staging for Keyman 12.0 release

### PR DESCRIPTION
**Pull request by darcywong00**
This is a staging branch for content pertaining to the Keyman 12.0 stable release

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
     
         <div class="section section-announcement">
           <div class='wrapper'>
-            <p><a href='/12/'>New: Keyman 12.0 is now in Beta!</a></p>
+            <p><a href='/12/'>New: Keyman 12.0 is now in released!</a></p>
```
Comment (mcdurdin) [inline index.php]:
```suggestion
            <p><a href='/12/'>New: Keyman 12.0 is now available!</a></p>
```

File: 12/index.php [hunk 2]
```diff
@@ -7,51 +7,50 @@
...
+<h1>Keyman 12.0 Stable</h1>
+<p class="red">7 October 2019</p>
+<li>Keyman 12.0 is now officially released! Find the 12.0 download for your platform below.</li>
 <br>
 
 <h3>What's New?</h3>
 
 <ul>
   <li>Predictive Text for Android and iOS</li>
+  <li>Create lexical model packages with Keyman Developer</li>
```
Comment (mcdurdin) [inline 12/index.php]:
```suggestion
  <li>Predictive Text for mobile phones and tablets &mdash; Android, iPhone and iPad</li>
  <li>Create predictive text lexical model packages for any language with Keyman Developer</li>
  <li>Keyman Developer projects simplified and more streamlined</li>
```

**Comment by darcywong00** [comment]
TODO: add "Donate" banner

**Comment by mcdurdin** [comment]
I've reworked the landing page for /12 to make it more feature-focused and simpler. Let me know what you think, y'all.

keyman com local_12_

**Pull request merged**